### PR TITLE
Cleaned up client class

### PR DESCRIPTION
**Pull request by ghost**
The client class had some bad params. Fixed those up. Added the setup_virsh
param to allow for configuring the ceph-virsh secret.

### CODE REVIEW

File: manifests/client.pp [hunk 1]
```diff
@@ -1,12 +1,13 @@
 class cephdeploy::client(
-  $user        = 'cephdeploy',
-  $primary_mon = 'kraken',
-  $pass        = hiera('ceph_deploy_password'),
+  $ceph_deploy_user = hiera('ceph_deploy_user'),
```
Comment (rickerc) [inline manifests/client.pp:2]: Isn't it considered bad form / unnecessary to have the explicit hiera call like that with scenario_node_terminus?

instead just a

$ceph_deploy_user = some_default_value

(ditto for all the others like that)

Comment (ghost) [inline manifests/client.pp:2]: it is bad form. however the rest of the module is in this format and until i get a chance to go rework it i want to have it uniform.

Comment (markvoelker) [inline manifests/client.pp:2]: Since it's relatively harmless, I can live with that for now.

**Comment by markvoelker** [comment]
+1

**Comment by rickerc** [comment]
Bug #1288307 filed so we don't forget to clean up

+1

**Pull request merged**